kimchi/src/linearization.rs

```diff
@@ -28,9 +29,9 @@ pub fn constraints_expr<F: FftField + SquareRootField>(
     let mut powers_of_alpha = Alphas::<F>::default();
 
     // gates
-    let highest_constraints = VarbaseMul::<F>::CONSTRAINTS;
+    let highest_constraints = Instruction::<F>::CONSTRAINTS;
     powers_of_alpha.register(
-        ArgumentType::Gate(GateType::VarBaseMul),
+        ArgumentType::Gate(GateType::CairoInstruction),
         highest_constraints,
     );
```

Comment:
can we do it like the lookup stuff? This way if we don't use cairo the alpha ranges are not affected

Reply:
Good point, I will give it a thought